_layouts/default.html

```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE html>
-<html lang="{{ page.lang | default: site.lang | default: "en" }}">
+<html lang="{{ page.lang | default: site.lang | default: 'en' }}">
 
 	{% include head.html %}
 
 	<body>
 		{% include navbar.html %}
 
-		<main id="main" role="main">
+		<main id="main">
```

Comment:
The `role` attribute is here, and should remain, so we support a wider range of assistive technologies, some of which recognize the `role` attribute and other ARIA attributes but do not recognize the `main` element in HTML5. It is redundant, but that's not necessarily a bad thing. :)